packages/core/documentation/src/components/ProductDocumentModal.vue

```diff
@@ -94,15 +95,40 @@
         />
       </template>
 
-      <template #action-buttons>
-        <div class="action-buttons">
+      <template #footer-actions>
+        <KButton
+          v-if="editing"
+          appearance="danger"
+          class="edit-documentation-delete-button"
+          data-testid="edit-documentation-delete-button"
+          :disabled="actionPending"
+          @click="emit('delete')"
+        >
+          <template
```

Comment:
Just drop the icon in the default button slot with the v-if

Reply:
Done